lms/static/scripts/ui-playground/components/EditAssignment.tsx

```diff
@@ -229,13 +224,13 @@ export default function EditAssignmentPage() {
                 <CardContent>
                   <CardActions>
                     <Button>Cancel</Button>
-                    <Button variant="primary">Continue</Button>
+                    <Button variant="primary">Save settings</Button>
```

Comment:
Some of these sketches use "Save settings" and others use just "Save". I assume that is an intentional difference of wording between the sketches?

Reply:
Correct. These are intended to show an assortment of options, cobbled together in different ways.